src/org/parosproxy/paros/control/MenuFileControl.java

```diff
@@ -511,12 +493,7 @@ public void sessionOpened(File file, Exception e) {
 
     @Override
     public void sessionSaved(Exception e) {
-        if (e == null) {
-            // ZAP: Set the title consistently
-            setTitle();
-            //File file = new File(model.getSession().getFileName());
-            //view.getMainFrame().setTitle(file.getName().replaceAll(".session\\z","") + " - " + Constant.PROGRAM_NAME);
-        } else {
+        if (e != null) {

```

Comment:
Can someone validate that this works out the same? From my PoV is does, but a second opinion is warranted.